core/src/migrations/000061-addImportStartedAt.ts

```diff
@@ -0,0 +1,12 @@
+export default {
+  up: async function (migration, DataTypes) {
+    await migration.addColumn("imports", "startedAt", {
+      type: DataTypes.DATE,
+      allowNull: true,
+    });
+  },
```

Comment:
We don't need to backfill this migration: either you are in the pending state and your `startedAt` is null, or you are already associated and have a `profileId` + related timestamp.